crates/taker/src/routes.rs

```diff
@@ -94,6 +98,8 @@ pub async fn feed(
 
         yield Event::json(&identity).event("identity");
 
+        yield Event::json(&environment).event("environment");
```

Comment:
🤔 This seems weird. Is there no alternative to this?

It's also questionable to rely on the environment variable to determine whether importing a seed is possible or not. Perhaps a better approach would be to selectively mount the route depending on the platform we are targeting? This would mean we would have a slightly different binary for `Umbrel`.

Reply:
Having to build different binaries just for that seems to be an overkill. I think one alternative to the environment variable, could be to use the app-seed flag. If set it means ItchySats is not managing the seed and hence will not support exporting or importing. It's implicitly what the umbrel environment is saying, but I guess that way its clearer.

I will adapt.